data/json/furniture_and_terrain/furniture_vitrified.json

```
@@ -0,0 +1,880 @@
+      "sound": "smash!",
+      "sound_fail": "whump.",
+      "items": [ { "item": "black_glass_shard", "count": [ 40, 80 ] } ]
+    },
+    "shoot": { "reduce_damage": [ 5, 20 ], "reduce_damage_laser": [ 5, 20 ], "destroy_damage": [ 15, 45 ] }
+  },
+  {
+    "type": "furniture",
+    "id": "f_vitrified_counter",
+    "name": "black glass counter",
+    "description": "A thick cube of black glass.  On the surface, fused in place, are statues of utensils and miscellany.",
+    "symbol": "#",
+    "color": "dark_gray",
+    "move_cost_mod": 2,
+    "coverage": 60,
+    "required_str": 10,
+    "flags": [ "TRANSPARENT", "MOUNTABLE", "SHORT", "FLAT_SURF" ],
+    "connect_groups": "COUNTER",
+    "connects_to": "COUNTER",
+    "bash": {
+      "str_min": 12,
+      "str_max": 40,
+      "sound": "smash!",
+      "sound_fail": "whump.",
+      "items": [ { "item": "black_glass_shard", "count": [ 40, 80 ] } ]
+    },
+    "examine_action": "workbench",
+    "workbench": { "multiplier": 1.1, "mass": 200000, "volume": "75L" }
+  },
+  {
+    "type": "furniture",
+    "id": "f_vitrified_table",
+    "name": "black glass table",
+    "description": "A slab of smooth black glass on four stout legs.  In a few places, there is shallowly rounded mass, like liquid spilled on the surface.",
+    "symbol": "T",
+    "color": "dark_gray",
+    "move_cost_mod": 2,
+    "coverage": 50,
+    "required_str": 5,
+    "flags": [ "TRANSPARENT", "MOUNTABLE", "SHORT", "FLAT_SURF", "PLACE_ITEM" ],
+    "deconstruct": { "items": [ { "item": "black_glass_shard", "count": [ 40, 80 ] } ] },
+    "bash": {
+      "str_min": 12,
+      "str_max": 50,
+      "sound": "smash!",
+      "sound_fail": "clink.",
+      "items": [ { "item": "black_glass_shard", "count": [ 40, 80 ] } ]
+    },
+    "examine_action": "workbench",
+    "workbench": { "multiplier": 1.1, "mass": 200000, "volume": "75L" }
+  },
+  {
+    "type": "furniture",
+    "id": "f_vitrified_birdbath",
+    "name": "black glass birdbath",
+    "description": "A glass pedestal and bowl.  Inside, frozen in the midst of washing itself in the black sand, is the delicate figure of a bird.",
+    "symbol": "o",
+    "color": "light_gray",
+    "move_cost_mod": -1,
+    "required_str": 10,
+    "flags": [ "PLACE_ITEM", "BLOCKSDOOR", "MINEABLE", "LIQUIDCONT", "TRANSPARENT" ],
+    "bash": {
+      "str_min": 16,
+      "str_max": 40,
+      "sound": "smash!",
+      "sound_fail": "clunk.",
+      "items": [ { "item": "black_glass_shard", "count": [ 40, 80 ] } ]
+    }
+  },
+  {
+    "type": "furniture",
+    "id": "f_vitrified_shower",
+    "name": "black glass shower",
+    "symbol": "~",
+    "description": "From the head of this shower descend a dozen streams of thin black glass, with frozen droplets strung from them like pearls.",
+    "color": "dark_gray",
+    "move_cost_mod": 0,
+    "coverage": 35,
+    "required_str": -1,
+    "flags": [ "TRANSPARENT", "FLAMMABLE_HARD", "CONTAINER", "PLACE_ITEM", "BLOCKSDOOR" ],
```

```suggestion
    "flags": [ "TRANSPARENT", "FLAMMABLE_HARD", "CONTAINER", "PLACE_ITEM", "BLOCKSDOOR" ],
```
```suggestion
    "flags": [ "TRANSPARENT", "CONTAINER", "PLACE_ITEM", "BLOCKSDOOR" ],
```
Not sure why this would be flammable and none of the rest are.